joebvp/joebgoodies.py

```diff
@@ -261,28 +194,14 @@ def wherebetween(testval,lim1,lim2):
 	result=np.where((testval>lim1)&(testval<lim2))[0]
 	return result
 
-# '''
-# ### Return sorted indices of matched coordinates from a list
-# def coordmatch(ralist,declist,ra,dec,tol=0.0001):
-#     radiff=np.abs(ralist-ra)
-#     decdiff=np.abs(declist-dec)
-#     match=where((radiff<=tol)&(decdiff<=tol))[0]
-#     if len(match)>1:
-#         seps=np.zeros(len(match))
-#         for i in range(len(match)):
-#             seps[i]=angsep(ralist[match[i]],declist[match[i]],ra,dec)
-#         order=np.argsort(seps)
-#         return match[order]
-#     else:
-#         return np.where((radiff<=tol)&(decdiff<=tol))[0]
-#
-# ### Estimate total H I mass given flat line profile
-# def himass(dist,S,dv):
-# 	'''
-# 	Units: [dist] = Mpc, [S] = Jy, [dv] = km/s
-# 	'''
-# 	return 2.356e5*dist**2*S*dv
-# '''
+
+### Estimate total H I mass given flat line profile
+def himass(dist,S,dv):
+ 	'''
+ 	Units: [dist] = Mpc, [S] = Jy, [dv] = km/s
+ 	'''
+ 	return 2.356e5*dist**2*S*dv
+
```

Comment:
Why is this here?

Reply:
Holdover from the code from which I had pasted a number of these routines.  Removed now.